error-messages/user.messages.js

```diff
@@ -0,0 +1,42 @@
+const CONFIRMATION_ERROR = [
```

Comment:
Please. remove these translations because now we don't return translations from backend anymore. We just return uppercase error message. Also, please, make error messages a little bit more informative, provide a clear reason, why the error has happened. `Oops! Something went wrong!` is not informative for the client